### PR TITLE
TRA-1959 nodejs fixes

### PR DESCRIPTION
**Pull request by RamiBerm**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import {getExpressMiddleware} from "./plugins/express-plugin";
-import {lowerCaseObjectKeys} from "./utils";
+import {formatRequestHeaders} from "./utils";
 import {UP9HttpConnector} from "./http-connector";
 
 const POLL_INTERVAL_MS = 5000;
```
Comment (gadotroee) [inline src/index.ts:5]: I think the name here is weird, maybe it is UP9 sender or something but it does not monitor,
not super critical but worth to think about it

File: src/index.ts [hunk 2]
```diff
@@ -79,10 +80,14 @@ class UP9Monitor {
     }
 
     requestLogger = (httpModule, protocol) => {
```
Comment (gadotroee) [inline src/index.ts]: I don't understand where do you use this "requestLogger"

File: src/index.ts [hunk 3]
```diff
@@ -79,10 +80,14 @@ class UP9Monitor {
...
-        let original = httpModule.request
-        httpModule.request = (request, callback) => {
+        httpModule.request = this.getHttpTappedFunc(httpModule.request, protocol);
```
Comment (gadotroee) [inline src/index.ts:83]: does it mean we only capture requests when use do the "get" or the "request" that used? (I guess there is also "put" / "post"..?)

File: src/index.ts (hunk 2 above)
Comment (RamiBerm) [reply src/index.ts]: It is not logged, it overrides http/https modules with a tapped version, ill rename to make more clear

File: src/index.ts (hunk 3 above)
Comment (RamiBerm) [reply src/index.ts:83]: .get is a quick method for GET requests, there are no .post/.patch etc
.request is used for all other methods

Comment (gadotroee) [reply src/index.ts:83]: weird that they do not have it

**Review by gadotroee — APPROVED** [review]

**Pull request merged**